src/lib/replication/repl_dev/raft_repl_dev.h

```diff
@@ -35,6 +35,11 @@ using raft_cluster_config_ptr_t = nuraft::ptr< nuraft::cluster_config >;
 
 ENUM(repl_dev_stage_t, uint8_t, INIT, ACTIVE, DESTROYING, DESTROYED, PERMANENT_DESTROYED);
 
+struct replace_members_ctx {
+    std::array< uint8_t, 16 > out;
```

Comment:
can we put comment for out/in, that it is replica_id_t, or can we just name them out_replica_id, in_replica_id

Reply:
Ack